README.md

```diff
@@ -1,13 +1,50 @@
-# bookstore
+### Bookstore application
```

Comment:
why do you have three # for this header here and two for other ones? The more hashes, the smaller the header level

Reply:
++, updated